### ISSUE TITLE
Foundry V10 Data Object Error

### ISSUE THREAD

**Issue by devindoherty** [kid2407/PerspectiveTokenScaler#1]
Hi there! Love the idea of this module for a little old school adventure game I'm running. I don't know if you're still maintaining this, but I got this error message in V10:

```
commons.js:1703 Error: You are accessing the DrawingDocument#data object which is no longer used. Since V10 the Document class and its contained DataModel are merged into a combined data structure. You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1692)
    at Function._logV10CompatibilityWarning (commons.js:6642)
    at DrawingDocument.get data [as data] (commons.js:6542)
    at PerspectiveTokenScaler.js:6
    at Map.filter (commons.js:3244)
    at Object.handleUpdate [as fn] (PerspectiveTokenScaler.js:6)
    at Function.#call (foundry.js:724)
    at Function.call (foundry.js:706)
    at ClientDatabaseBackend._preUpdateDocumentArray (foundry.js:12736)
    at async ClientDatabaseBackend._updateEmbeddedDocuments (foundry.js:12661)
    at async Function.updateDocuments (commons.js:6153)

```

**Comment by kid2407** [comment]
Good thing: Is is not an error, but a deprecation warning.

More good news: I can totally fix that so there is one less warning and it becomes future-proof. I can't guarantee when I'm able to, but I'll try to get it done within a reasonable time frame.

**Comment by devindoherty** [comment]
Amazing! I was trying to use the module and set it up according to the gif on the repo, but it didn't seem to be working. I assumed this had something to do with it.

**Comment by kid2407** [comment]
If you like, could you export the scene and send me the file / attach it here? Perhaps I can tell where things go wrong.

**Comment by devindoherty** [comment]
Do you only need the JSON? 

[fvtt-Scene-the-roost-courtroom-(copy)txt.txt](https://github.com/kid2407/PerspectiveTokenScaler/files/10529181/fvtt-Scene-the-roost-courtroom-.copy.txt.txt)

![image](https://user-images.githubusercontent.com/21369979/215309373-5e3ab610-bb6b-4293-8505-eee479f4543a.png)

I believe it's all setup correctly, but I could be mistaken.

This is the full console log:

[full_log_perspective.txt](https://github.com/kid2407/PerspectiveTokenScaler/files/10529345/full_log_perspective.txt)

**Comment by kid2407** [comment]
Yes, the scene export is all I'll need (the log helps too in case something strange happens / goes on). Will look into it an dget back to you.

**Comment by kid2407** [comment]
Should be fixed with Version 1.1.0, built and published via the package manager.
